samples/trumpify/trump.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import time
+from threading import Thread
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
+
+from daemo.client import Client
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
```

Comment:
How often does the user need to download a new file?

On Tue, Jul 19, 2016 at 9:57 AM, Durim Morina <notifications@github.com>
wrote:

> In samples/trumpify/trump.py
> <https://github.com/crowdresearch/daemo-api-client/pull/1#discussion_r71378983>
> :
>
> > +from threading import Thread
> > +
> > +from twitter import *
> > +
> > +sys.path.append(os.path.abspath('../../'))
> > +
> > +from daemo.client import Client
> > +
> > +CREDENTIALS_FILE = 'credentials.json'
> > +
> > +PROJECT_ID = -1
> > +
> > +TW_CONSUMER_KEY = ''
> > +TW_CONSUMER_SECRET = ''
> > +TW_ACCESS_TOKEN = ''
> > +TW_ACCESS_TOKEN_SECRET = ''
>
> The thing about oauth2 refresh tokens is that they change, and that
> wouldn't quite work with env variables because we would have to update
> those variables, we would need some sort of storage
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/crowdresearch/daemo-api-client/pull/1/files/82b61b3e4eebde74a63b06dc7d814226f2e0c914#r71378983>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPNxbF6SGTxndQaQ-pT0L4mUmADNuJKks5qXQHqgaJpZM4I_odP>
> .
>